index.html

```diff
@@ -1483,6 +1483,21 @@
 
 </div>
 
+## Non-Retaliation {#non-retaliation}
+
+<div class="practice">
+  <span class="practicelab" id="principle-do-not-retaliate">
+    [=Parties=] must not retaliate against [=people=] who protect their [=data=] against
```

Comment:
Can this also apply to party on party retaliation?

Parties must not retaliate against people who protect their own data, or against other parties that protect people's data...